backend/models/issues.py

```diff
@@ -36,9 +36,16 @@ def delete(self):
         if not os.listdir(directory_path):
             os.rmdir(directory_path)
 
+    def get_full_thumbnail_url(self):
+        url = self.get_thumbnail_url()
+        if current_app.config.get('MEDIA_URL'):
+            return '%s%s' % (current_app.config['MEDIA_URL'], url)
+        return '%s%s' % ('/media/', url)
```

Comment:
It doesn't make sense to use formatting here, concatenation is enough:
```
       if current_app.config.get('MEDIA_URL'):
            return current_app.config['MEDIA_URL'] + url
       return '/media/' + url
```

Reply:
fixed